dts/arm/st/l4/stm32l4.dtsi

```diff
@@ -234,6 +234,13 @@
 				label = "PWM_2";
 				#pwm-cells = <3>;
 			};
+
+			hwtrig2: hwtrig {
```

Comment:
because this is a timer based device, it should be part of the `timersN` node, same as PWM.

Reply:
This is what I did. Am I missing something?